sunpy/map/mapbase.py

```diff
@@ -195,21 +199,19 @@ def __init__(self, data, header, plot_settings=None, **kwargs):
         self._validate_meta()
         self._shift = SpatialPair(0 * u.arcsec, 0 * u.arcsec)
 
-        if self.dtype == np.uint8:
-            norm = None
-        else:
-            # Put import here to reduce sunpy.map import time
-            from matplotlib import colors
-            norm = colors.Normalize()
-
         # Visualization attributes
-        self.plot_settings = {'cmap': 'gray',
-                              'norm': norm,
-                              'interpolation': 'nearest',
-                              'origin': 'lower'
-                              }
+        self.cmap = cmap
+        self.norm = norm
         if plot_settings:
-            self.plot_settings.update(plot_settings)
+            self.plot_settings = plot_settings
```

Comment:
Move this just outside the `if` block. That way, the `plot_settings` attribute always exists.